packages/webdriverio/src/index.ts

```diff
@@ -179,6 +179,7 @@ export default class AxeBuilder {
   private get script(): string {
     return `
       ${this.axeSource}
+      axe.configure({ allowedOrigins: ['<unsafe_all_origins>'] })
```

Comment:
Any reason not to merge this with the `axe.configure()` call below?

Reply:
We set it set `axe.configure()` so that we can inject axe and get the results from all iframes https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#allowedorigins instead of just same origin iframes

Reply:
Yes.

My question is why we're not doing this:

```
axe.configure({
  allowedOrigins: [...],
  branding: { application: 'webdriverio' }
})
```

NBD tho, which is why I approved.